fab/inventory/softlayer

```diff
@@ -139,6 +148,9 @@ riak15
 
 [couchdb2:children]
 couch1
+couch2
```

Comment:
Should the proxy get moved to one of these?

Reply:
Yeah, good call. I'll do that next